app/views/home/_administration.html.erb

```diff
@@ -2,7 +2,7 @@
   <h2>Administration</h2>
   <ul>
     <li><%= link_to 'Requests dashboard', admin_index_path %></li>
-    <li><%= link_to "Broadcast messages", "#"%> </li>
+    <li><%= link_to "Broadcast messages", messages_path %> </li>
```

Comment:
I'm inclined to just ship it. I'm not clear on the different authorization rules we're imagining anyway.